src/formatters/index.js

```diff
@@ -46,8 +46,23 @@ export function generateStats(data: Array<CheckstyleItem>): Stats {
 }
 
 export function formatOutput(format: string, data: Array<CheckstyleItem>) {
-  if (!{}.hasOwnProperty.call(formatters, format)) {
-    throw new Error(`Formatter with name '${format}' does not exist.`)
+  let formatter
+  if (/^require:/.test(format)) {
```

Comment:
I did it explicitly because `require` is very slow and most use cases is probably going to use an internal one. Also being explicitly is good thing :v: